products/views.py

```diff
@@ -147,3 +147,20 @@ def get(self, request):
         } for product in products[offset:limit]]
 
         return JsonResponse({'results' : results, 'total_count': total_count}, status =200)
+
+class LikeToggleView(View):
+    @signin_decorator
+    def post(self, request, product_id):
+        try:
+            product  = Product.objects.get(id=product_id)
+            like     = Like.objects.filter(product_id=product.id, user_id=request.user.id)
+            
+            if like:
+                like.delete()
+                return JsonResponse({'massage':"ToggleSuccess"}, status=200)
+            else:
+                Like.objects.create(product_id=product.id, user_id=request.user.id)
+                return JsonResponse({'massage':"ToggleSuccess"}, status=200)
+                
+        except Product.DoesNotExist:
+            return JsonResponse({'massage':"DoesNotExist"}, status=401)
```

Comment:
Toggle은 페이지를 기준으로 만들기 때문에 생기는 네이밍입니다.
자원 기준으로 Like 만들어주세요~

```suggestion
class LikeView(View):
    @signin_decorator
    def post(self, request, product_id):
        try:
            like     = Like.objects.filter(product_id=product.id, user_id=request.user.id)
            
            if like:
                like.delete()
                return JsonResponse({'massage':"ToggleSuccess"}, status=200)
            else:
                Like.objects.create(product_id=product.id, user_id=request.user.id)
                return JsonResponse({'massage':"ToggleSuccess"}, status=200)
                
        except Product.DoesNotExist:
            return JsonResponse({'massage':"DoesNotExist"}, status=401)
```